locals.tf

```diff
@@ -0,0 +1,21 @@
+locals {
+  unauthorized_api_calls_base_exclusions = [
+    "($.sourceIPAddress != \"delivery.logs.amazonaws.com\")",
+    "($.eventName != \"HeadBucket\")"
+  ]
+
+  unauthorized_api_calls_extra_exclusions = [
+    for svc in var.unauthorized_api_calls_extra_excluded_services :
+    "($.eventSource != \"${svc}\")"
```

Comment:
why not on one line? If separating to newline, then indent the second line to make it clear it's part of the loop.